### ISSUE TITLE
`dbGetQuery()` returns no columns for INSERT ... RETURNING on temporary tables

### ISSUE THREAD

**Issue by mgirlich** [r-dbi/RSQLite#380]
Temporary tables return a tibble with the correct number of rows but with no columns

``` r
x <- tibble::tibble(
  x = 1:3,
  y = runif(3)
)

con <- DBI::dbConnect(RSQLite::SQLite(), ":memory:", create = TRUE)
DBI::dbWriteTable(con, "tbl", x)
DBI::dbGetQuery(
  con,
  "INSERT INTO tbl (`x`, `y`)
   VALUES (100, 1)
   RETURNING *",
  immediate = TRUE
)
#>     x y
#> 1 100 1

DBI::dbWriteTable(con, "tmp_tbl", x, temporary = TRUE)
DBI::dbGetQuery(
  con,
  "INSERT INTO tmp_tbl (`x`, `y`)
   VALUES (100, 1)
   RETURNING *",
  immediate = TRUE
)
#> data frame with 0 columns and 4 rows
```

<sup>Created on 2021-08-05 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by krlmlr** [comment]
Thanks. This appears to be an [undocumented](https://www.sqlite.org/lang_returning.html) restriction of SQLite:

```
$ echo "
CREATE TEMPORARY TABLE tmp_tbl (x, y)
GO
SELECT COUNT(*) FROM tmp_tbl
GO
INSERT INTO tmp_tbl (x, y) VALUES (100, 1) RETURNING *
GO" | sqlite3
0
$ echo "
CREATE TABLE tmp_tbl (x, y)
GO
SELECT COUNT(*) FROM tmp_tbl
GO
INSERT INTO tmp_tbl (x, y) VALUES (100, 1) RETURNING *
GO" | sqlite3
0
100|1
```

Can you please raise this upstream?

**Comment by mgirlich** [comment]
I just found out that this was fixed in SQLite 3.36.0 and therefore works with the dev version of RSQLite 🥳

**Comment by krlmlr** [comment]
I should be installing packages from my r-universe :sweat_smile:
